substrate/frame/identity/src/lib.rs

```diff
@@ -226,6 +246,11 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxUsernameLength: Get<u32>;
 
+		/// A set of helper functions for benchmarking.
+		/// The default configuration `()` uses the `SR25519` signature schema.
+		#[cfg(feature = "runtime-benchmarks")]
+		type BenchmarkHelper: BenchmarkHelper<Self::SigningPublicKey, Self::OffchainSignature>;
```

Comment:
if you wnat to make it backwards compatible, convert this into a function:
```
fn benchmark_helper(message: &[u8]) -> (Vec<u8>, Vec<u8>) {
     		let public = sp_io::crypto::sr25519_generate(0.into(), None);
		let signature = sp_runtime::MultiSignature::Sr25519
			sp_io::crypto::sr25519_sign(
				0.into(),
					&public.into_account().try_into().unwrap()
					message,
				)
				.unwrap(),
			);

	(public.encode(), signature.encode())
}
```

Then you need to decode this in the benchmark etc.

Reply:
Not sure how `IdentifyAccount` is related here? Because you need to encode/decode the public key and the signature.